### DISCUSSION TITLE
Parameter and configuration of cycle count

### DISCUSSION THREAD

**Discussion by riker65** [syssi/esphome-jk-bms#324] [General]
Hi
anyone knowing how a cycle is defined?

after discharging I now have fully charged the cells - at least nominal.

But cycle count does not increase.

**Comment by syssi** [comment]
Let's assume the capacity of your battery bank is 280 Ah. If you discharge your battery by 280 Ah and charge the battery again by 280 Ah the cycle count should increase by one. May be a few Ah are still missing.

**Comment by riker65** [comment]
ok thanks, but for discharge, how far to identify a discharge? identified by voltage discharge limit set or Ah consumed? little unclear to me.
because  bms is set to 280 AH cells, showing capacity % 100 % and AH also 280 already but cycle did not increase

**Comment by riker65** [comment]
over voltage was triggered already as well

**Reply by syssi** [reply]
Be patient. ;-) If you discharge + charge your battery 28x by 10 Ah the cycle count should increase by 1 too. I guess your BMS assumes the full cycle isn't met yet.

**Reply by catalinbordan** [reply]
If I remember correctly, you have total capacity discharged (lifetime). this divided with your battery capacity it will tell you the nr of cycles.
if you change the battery capacity, the total capacity discharged, number of cycles it will be reset.
